### ISSUE TITLE
Probar calendario

### ISSUE THREAD

**Issue by pollitofive** [letme-ar/shelter#6]
Ya esta listo para que pruebes el calendario.

**Comment by msarfernandez** [comment]
Bueno, estuve mirando... Te había consultado por mail esto que no entendí: "Tengo un problema porque por mas que no cambie la reserva en la base, no me termina de quedar bien la reserva en el calendario. Estoy pensando seriamente en refrescar la pagina si se equivoca." 
Por lo que estuve chequeando, veo que al final se actualiza cuando te equivocas (de nuevo, no entendí por qué lo harías); y como que suceden cosas que te marean un poco. 
Por ejemplo, en este momento me paro en la semana próxima (miércoles 18 y jueves 19), y cargo dos reservas, una cada día. Cuando arrastro la del jueves hacia encima de la del miércoles, primero la asigna, luego me tira el cartel y después actualiza y me para de vuelta a la semana de hoy... me explico?
Habría que ajustar algunas cuestiones al respecto. Primero quizá entender por qué tiene que actualizar. No debería volver, y se me ocurre que debería validarlo en el momento... ni siquiera asignarlo antes. Cómo lo ves?

**Comment by pollitofive** [comment]
Ese es el problema: Que primero la asigna. Cuando arrastras una reserva hacia otra, la asigna y no puedo volver atras. Siempre hablando de la parte del front, porque yo valido y no actualizo en la base porque me dice que hay otra cargada. Pero no hay forma de que vuelva a poner todo como estaba antes, cosa que si ocurre cuando vos "Agrandas" una reserva y hay otra. Fijate que en ese caso puedo volver todo para atras sin que haya que refrescar la pagina

**Comment by msarfernandez** [comment]
Claro... ahí vi que cuando agrandás, está todo bien... habría que buscarle la vuelta che... o parchar que si estás en otra semana, no te traiga a la actual al recargar, me explico?

**Comment by msarfernandez** [comment]
No se puede traer todas las reservas de esa semana que se está mostrando y validar por JS en el momento? Me explico?
